packages/mui-material/src/Popover/Popover.d.ts

```diff
@@ -84,6 +84,13 @@ export interface PopoverProps
    * so it's simply `document.body` most of the time.
    */
   container?: ModalProps['container'];
+  /**
+   * Disables the margin threshold so that when scrolled the popover stays in the correct position
+   * By default, false for backward compatibility.
+   *
+   * @default false
+   */
+  disableMarginThreshold?: boolean;
```

Comment:
The name of this prop doesn't say much about it's behavior, and more about implementation. I find it hard to figure out what's its purpose without reading the description (and even there only the "when scrolled the popover stays in the correct position" makes sense).

Reply:
Do you think this is a description issue? What would you think about: "Disable margin threshold, removing the constraints of where the popover can appear"

I couldn't think of a better name given that `marginThreshold` is the prop being bypassed if `disableMarginThreshold` is `true`

Reply:
How about allowing `null` in `marginThreshold` instead and documenting it well?

Reply:
Good idea! I refactored and pushed this change